sdk/resourcemanager/azure-resourcemanager-resources/src/main/java/com/azure/resourcemanager/resources/fluentcore/arm/implementation/AzureConfigurableImpl.java

```diff
@@ -32,18 +31,17 @@
     implements AzureConfigurable<T> {
     private HttpClient httpClient;
     private HttpLogOptions httpLogOptions;
-    private List<HttpPipelinePolicy> policies;
-    private List<String> scopes;
+    private final List<HttpPipelinePolicy> policies;
+    private final List<String> scopes;
     private RetryPolicy retryPolicy;
     private RetryOptions retryOptions;
     private Configuration configuration;
-    private List<TokenCredential> tokens;
+    private final List<TokenCredential> tokens;
 
     protected AzureConfigurableImpl() {
         policies = new ArrayList<>();
         scopes = new ArrayList<>();
         tokens = new ArrayList<>();
-        retryPolicy = new RetryPolicy("Retry-After", ChronoUnit.SECONDS);
```

Comment:
Delay the default to `HttpPipelineProvider`.

So that code here would now have effect https://github.com/Azure/azure-sdk-for-java/blob/90775e584eb5bae2f8bb929090440ac24ddb6d95/sdk/resourcemanager/azure-resourcemanager-resources/src/main/java/com/azure/resourcemanager/resources/fluentcore/arm/implementation/AzureConfigurableImpl.java#L142-L144
(previously `retryPolicy` variable will not be null)